apps/web/src/interfaces/index.ts

```diff
@@ -133,3 +133,27 @@ export type EventGuest = {
 /** End of RO3 */
 
 // web specific interfaces below
+
+export const SUPPORTED_TIME_FORMATS = [
+  'h:mm a',
+  'h:mm A',
+  'hh:mm a',
+  'hh:mm A',
+  'HH:mm',
+];
+
+export interface HeatMapData {
+  yTimesHeatMapLabelsArray: string[];
+  xDaysHeatMapLabelsArray: string[];
+  xDaysFormattedToSlicedDateString: string[];
+  heatMap2dArray: number[][];
+}
+
+export interface ScheduleSelectorData {
+  scheduleData: Array<Date>;
```

Comment:
NIt: Same as Date[], id prefer this notation because its familiar to how other langs declare types and typically <> are for generics

Reply:
What is NLT? Okay, you're right, Ill make that change.